app/services/state_file/state_information_service.rb

```diff
@@ -169,6 +174,7 @@ def state_intake_class_names
         tax_payment_info_url: "https://www.nj.gov/treasury/taxation/payments-notices.shtml",
         tax_payment_url: "https://www.nj.gov/treasury/taxation/payments-notices.shtml",
         tax_refund_url: "https://www.tax.ny.gov/pit/file/refund.htm",
+        department_of_taxation: "",
```

Comment:
should we ping NJ team about this change

Reply:
i left a bunch of them blank because i guess this story is only for AZ changes. i will double check with tiffany but we should have follow up stories for this content for other states

Reply:
I think we still need to check in with NJ about this - especially since this field is re-used in a couple places